data-sets-tests/src/test/java/org/zowe/unix/files/tests/UnixFilesDeleteFileTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2019
+ */
+package org.zowe.unix.files.tests;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.apache.http.HttpStatus;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.zowe.api.common.errors.ApiError;
+import org.zowe.tests.AbstractHttpIntegrationTest;
+import org.zowe.unix.files.exceptions.FileNotFoundException;
+import org.zowe.unix.files.exceptions.NotAEmptyDirectoryException;
+import org.zowe.unix.files.exceptions.PermissionDeniedFileException;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+
+public class UnixFilesDeleteFileTest extends AbstractHttpIntegrationTest {
+
+    static final String UNIX_FILES_ENDPOINT = "unixfiles";
+	static final String TEST_DIRECTORY = System.getProperty("server.test.directory");
+    
+    @BeforeClass
+    public static void setUpEndpoint() throws Exception {
+        RestAssured.basePath = UNIX_FILES_ENDPOINT;
+    }
+    
+    @Test
+    public void testDeleteUnixFileContent() throws Exception {
+    	final String fvtDeleteFile = TEST_DIRECTORY + "/deleteTestDirectoryAccess/deleteFileWithWritePermission";
+        
+        RestAssured.given().when().delete(fvtDeleteFile)
```

Comment:
I think we should validate that a user gets a not found exception when a GET request is sent after this

Reply:
extended delete test - added get after delete to verify further